java-test-samples/apigw-lambda-ddb/README.md

```diff
@@ -68,6 +68,36 @@ To test the endpoint first send data using the following command. Be sure to upd
 curl -X POST https://COPYfromAPIGateway/dev/ticket -H "Content-Type: application/json" -d '{"userId": "231deb432f3dd","description": "My monitor is broken."}' 
 ```
 
+## Automated Tests
+The source code for this sample includes automated tests.
+### Running Automated Tests
+These tests are run while packaging the lambda function using `mvn clean package` or can be run standalone using `mvn test`.
+### Automated Test Framework
+[Junit 5](https://junit.org/junit5/) is the primary test framework used to write these tests. A few other libraries and frameworks are used depending on the test case pattern. Please see below. 
+### Types of Automated Tests
+The sample includes the following tests:
+
+#### Unit Test (TicketFunctionMockTest.java)
```

Comment:
@muraria Can you link the test file from here ? You can use relative path and make the name of the java class a link to the actual class. Same is applicable for other tests.